docs/src/api.md

```diff
@@ -64,7 +64,6 @@ fit
 StatsBase.deviance
 GLM.dispersion
 GLM.ftest
-GLM.installbeta!
```

Comment:
If this was documented as part of the API, should we add a simple deprecation like
```julia
@deprecate installbeta!(p) (p.beta0 .= p.delbeta) false
@deprecate installbeta!(p, f) (p.beta0 .= p.delbeta .* f) false
```
?

Reply:
I thought it might be okay since we are preparing for a breaking change anyway and I was not envisioning that anybody would be using it but [I was wrong](https://juliahub.com/ui/Search?q=installbeta&type=code). All add some deprecations.

Reply:
😬